README.md

```diff
@@ -9,7 +9,7 @@ products:
 description: Get started quickly with Express and PostgreSQL (apptemplate)
 ---
 
-# Welcome to the Express and PostgreSQL template
+# WebAPI with Database using Node.js and PostgreSQL
```

Comment:
I would prefer something more descriptive, like "Application Template for WebAPI and Database using Node.js and PostgreSQL"